dts/bindings/display/rocktech,rk043fn02h-ct.yaml

```diff
@@ -6,3 +6,7 @@ description: Rocktech LCD module with LED backlight and capacitive touch panel
 compatible: "rocktech,rk043fn02h-ct"
 
 include: base.yaml
+
+properties:
+    label:
+      required: true
```

Comment:
Is this still required if the relationship is managed through phandle references rather than the label?

Reply:
Yes we can remove that when switching to phandle.